### PR TITLE
Create kubecf+operator bundle files as an artifact

### PR DESCRIPTION
**Pull request by mook-as**
## Description
This makes it easier for QA to install the matching versions.

## Motivation and Context
For QA, they'd like to be sure they get the correct cf-operator for a given kubecf build without having to dig through the sources to find the file.  This is related to (but not the same as) #124.

Fixes jsc#CAP-1029.

## How Has This Been Tested?
Locally ran CI's [build.sh] and checked that the expected files are generated.  Will also trigger CI.

[build.sh]: https://github.com/SUSE/kubecf/blob/03408ebb8e373449f4ed896c6fc129147e46a8fd/.gitlab/pipelines/stages/build/build.sh

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code has security implications.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .gitlab/pipelines/stages/publish.sh [hunk 1]
```diff
@@ -3,5 +3,7 @@
 set -o errexit -o nounset
 
 mc config host add "s3" "https://s3.amazonaws.com" "${AWS_ACCESS_KEY}" "${AWS_SECRET_KEY}"
-mc cp output/kubecf-*.tgz "s3/scf-v3"
-echo "URL: https://scf-v3.s3.amazonaws.com/$(basename output/kubecf-*.tgz)"
+for file in output/kubecf-*.tgz ; do
```
Comment (f0rmiga) [inline .gitlab/pipelines/stages/publish.sh]: This is an unnecessary change. `kubecf-*` should match only one file under `output`. If we get more than one, something is wrong.

Comment (mook-as) [reply .gitlab/pipelines/stages/publish.sh]: Ah, yeah, I had made the bundle be called `kubecf-bundle-blah`, but then had to change it to `bundle-blah` because actually deploying it got annoying. Will fix this.

Comment (mook-as) [reply .gitlab/pipelines/stages/publish.sh]: Fixed in cb180aa56f8c159344d7888cb62c3976dae57d0d
